mapie/tests/test_time_series_regression.py

```diff
@@ -427,3 +453,58 @@ def test_consistent_class() -> None:
     np.testing.assert_allclose(y_pis_1[:, 0, 0], y_pis_2[:, 0, 0])
     np.testing.assert_allclose(y_pis_1[:, 1, 0], y_pis_2[:, 1, 0])
     np.testing.assert_allclose(y_pred_1, y_pred_2)
+
+
+def test_aci_method() -> None:
+    """
+    Test function for the "aci" (Adapted Conformal Inference) method
+    in a MapieTimeSeriesRegressor.
+    Additionally, it attempts to test the regressor with the "enbpi"
+    method, but this part is expected to raise an exception,
+    and it captures the exception without taking any action.
+    """
+    mapie_regressor = MapieTimeSeriesRegressor(method="aci")
+    mapie_regressor.fit(X, y)
+    mapie_regressor.predict(X, alpha=0.05)
+    mapie_regressor.adapt_conformal_inference(X, y, gamma=0.01)
+    with pytest.raises(
+        AttributeError,
+        match=r"This method can be called only with method='aci' *"
+    ):
+        mapie_regressor_enbpi = MapieTimeSeriesRegressor(method="enbpi")
+        mapie_regressor_enbpi.fit(X, y)
+        mapie_regressor_enbpi.adapt_conformal_inference(X, y, gamma=0.01)
+
+
+def test_aci_init_and_reset_alpha_dict() -> None:
+    """Test that `_get_alpha` resets all the values in the dictionary."""
+    mapie_ts_reg = MapieTimeSeriesRegressor(method="aci")
+    mapie_ts_reg._get_alpha()
+    np.testing.assert_equal(isinstance(mapie_ts_reg.current_alpha, dict), True)
+
+    mapie_ts_reg.current_alpha[0.05] = 0.45
+    mapie_ts_reg._get_alpha(reset=True)
+    np.testing.assert_equal(bool(mapie_ts_reg.current_alpha), False)
+
+
+def test_aci__get_alpha_with_unknown_alpha() -> None:
+    """
+    Test that the `adapt_conformal_inference` method initializes
+    a new value if alpha is seen for the first time.
+    """
+    mapie_ts_reg = MapieTimeSeriesRegressor(method="aci")
+    mapie_ts_reg.fit(X_toy, y_toy)
+    mapie_ts_reg.adapt_conformal_inference(X_toy, y_toy, gamma=0.1, alpha=0.2)
+    np.testing.assert_allclose(mapie_ts_reg.current_alpha[0.2], 0.3, rtol=1e-3)
```

Comment:
```suggestion
    np.testing.assert_allclose(mapie_ts_reg.current_alpha[0.2], 0.3, rtol=1e-3)


def test_deprecated_partial_fit_warning(method: str) -> None:
"""
Test that a warning is raised if use partial_fit
"""
mapie_ts_reg = MapieTimeSeriesRegressor(method='enbpi', cv=-1)
mapie_ts_reg.fit(X_toy, y_toy)
with pytest.warns(
    DeprecationWarning, match=r".*WARNING: Deprecated method.*"
):
    mapie_ts_reg = mapie_ts_reg.partial_fit(X_toy, y_toy)
```

Reply:
I will commit with GitHub (with previous suggested change). I just need you to add an indentation.

Reply:
```suggestion
    np.testing.assert_allclose(mapie_ts_reg.current_alpha[0.2], 0.3, rtol=1e-3)
def test_deprecated_partial_fit_warning(method: str) -> None:
    """
    Test that a warning is raised if use partial_fit
    """
    mapie_ts_reg = MapieTimeSeriesRegressor(method='enbpi', cv=-1)
    mapie_ts_reg.fit(X_toy, y_toy)
    with pytest.warns(
        DeprecationWarning, match=r".*WARNING: Deprecated method.*"
    ):
        mapie_ts_reg = mapie_ts_reg.partial_fit(X_toy, y_toy)
```